juju_verify/verifiers/ceph.py

```diff
@@ -79,8 +94,20 @@ def __hash__(self) -> int:
         return hash(self.__str__())
 
 
-class AvailabilityZone:
-    """Availability zone."""
+class CephTree:
+    """Ceph tree."""
+
+    SUPPORTED_REPLICATION_RULE = [
+        "region",
+        "datacenter",
+        "room",
+        "pod",
+        "pdu",
+        "row",
+        "rack",
+        "chassis",
+        "host",
+    ]
```

Comment:
These are all the crush rules except "root".  Maybe that could be stated in a comment right above line 100.

Reply:
I renamed the variable and add comment.